qa/rpc-tests/getblockexpanded.py

```diff
@@ -0,0 +1,260 @@
+        print("########### Add to mempool Certificate 2 with quality = 7 ##################")
+        quality = 7  
+        bwt_amount2 = Decimal("7")      
+        proof = mcTest.create_test_proof(
+            "sc1", scid_swapped, epoch_number, quality, MBTR_SC_FEE, FT_SC_FEE, epoch_cum_tree_hash, constant, [node1Addr], [bwt_amount2])
+
+        amount_cert_2 = [{"address": node1Addr, "amount": bwt_amount2}]
+
+        self.nodes[0].sc_send_certificate(scid, epoch_number, quality,
+            epoch_cum_tree_hash, proof, amount_cert_2, FT_SC_FEE, MBTR_SC_FEE, CERT_FEE)
+        self.sync_all()
+
+        #Mine a block
+        self.nodes[0].generate(1)[0]
+        self.sync_all()
+
+        #Mine a block with a new Certificate 3 with quality = 8
+        print("########### Mine a block with a new Certificate 3 with quality = 8 ##################")
+        quality = 8
+        bwt_amount3 = Decimal("7")      
+        proof = mcTest.create_test_proof(
+            "sc1", scid_swapped, epoch_number, quality, MBTR_SC_FEE, FT_SC_FEE, epoch_cum_tree_hash, constant, [node1Addr], [bwt_amount3])
+
+        amount_cert_3 = [{"address": node1Addr, "amount": bwt_amount3}]
+
+        cert3 = self.nodes[0].sc_send_certificate(scid, epoch_number, quality,
+            epoch_cum_tree_hash, proof, amount_cert_3, FT_SC_FEE, MBTR_SC_FEE, CERT_FEE)
+        self.sync_all()
+        cert3_block = self.nodes[0].generate(1)[0]
+        self.sync_all()
+
+        #Advance of 1 epoch
+        print("########### Advance of 1 epoch ##################")
+        self.nodes[0].generate(116)
+        self.sync_all()
+
+        epoch_number, epoch_cum_tree_hash = get_epoch_data(scid, self.nodes[0], EPOCH_LENGTH)
+        quality = 9
+        bwt_amount4 = Decimal("9")
+        proof = mcTest.create_test_proof(
+            "sc1", scid_swapped, epoch_number, quality, MBTR_SC_FEE, FT_SC_FEE, epoch_cum_tree_hash, constant, [node1Addr], [bwt_amount4])
+
+        amount_cert_4 = [{"address": node1Addr, "amount": bwt_amount4}]
+
+        self.nodes[0].sc_send_certificate(scid, epoch_number, quality,
+            epoch_cum_tree_hash, proof, amount_cert_4, FT_SC_FEE, MBTR_SC_FEE, CERT_FEE)
+
+        cert4_block = self.nodes[0].generate(1)[0]
+        self.sync_all()
+
+        #Enter in the next epoch
+        new_epoch_block = self.nodes[0].generate(1)[0]
+        self.sync_all()
+
+        rpcCertBlock = self.nodes[0].getblock(cert3_block, 2)
+        cert_3_json = {}
+        for cert in rpcCertBlock['cert']:
+            if (cert['txid'] == cert3):
+                cert_3_json = cert
+        assert_true(cert_3_json != {})
+        tipHeight = self.nodes[0].getblockcount()
+
+        #Test that we require -maturityheightindex=1 to run the getblockexpanded
+        try:
+            self.nodes[1].getblockexpanded("640")
+            assert(False)
+        except JSONRPCException as e:
+            errorString = e.error['message']
```

Comment:
`errorString` is not used in this scope, should we print it in the following debug log?